recipes-mender/mender/files/mender.service

```diff
@@ -6,6 +6,7 @@ Type=simple
 User=root
 Group=root
 ExecStartPre=/bin/mkdir -p -m 0700 /data/mender
+ExecStartPre=/bin/ln -s /etc/mender/tenant.conf /var/lib/mender/authtentoken
```

Comment:
Won't `ln` fail if the link already exists? Quote from systemd docs:

> If any of those commands (not prefixed with "-") fail, the rest are not executed and the unit is considered failed.

So the unit may only start on the first boot, but not thereafter.